tests/Tests/ORM/Functional/Ticket/GH11149/Locale.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\ORM\Functional\Ticket\GH11149;
+
+use Doctrine\ORM\Mapping as ORM;
+
+#[ORM\Entity]
+#[ORM\Table('gh11149_locale')]
+class Locale
+{
+    #[ORM\Id]
+    #[ORM\Column]
```

Comment:
Not sure if that's what you were referring to as a bug, but I think you need to specify a length here, for MariaDB.

Reply:
That's probably it: https://github.com/doctrine/dbal/blob/4.0.x/UPGRADE.md#bc-break-changes-in-handling-string-and-binary-columns

Reply:
Ah, right. I completely missed that. We should've done this on 2.19.x already I guess because having a column with length 255 doesn't really make sense anyway. 🙂 

I'll make a PR for 2.19 and prepare a new merge-up.

Reply:
#11393

Reply:
It's great if you do, but I'm not sure we should bother porting this every time, since we don't plan to ever add support for DBAL 4 on 2.x I'm myself guilty of fixing that same issue on other tests in merge PRs several times.